etc/bash/cs.sh

```diff
@@ -19,3 +19,8 @@ cd -
 cd TaskManager
 composer run-script cs
 cd -
+
+cd CompositeUi
+node_modules/.bin/eslint -c .eslintrc.js src/ --fix
+# node_modules/.bin/stylefmt -c .stylelintrc.js -r src/scss/**/*.scss
```

Comment:
It has a bug related with empty line between Sass imports and Sass variables so, it is not ready to use yet. I'm watching the Github issue related with this bug so, after the problem has been corrected, we'll uncomment this line